rules/tines_rules/tines_global_resource_destruction.py

```diff
@@ -0,0 +1,7 @@
+def rule(event):
+    return event.get('operation_name') == 'GlobalResourceDestruction'
+
+def title(event):
+    user = event.get('user_email')
+    tines_instance = event.get('p_source_label')
```

Comment:
Please use `deep_get()` with a `default=` kwarg to ensure that these have value.  I would recommend the same for the `rule()` function.